backend/src/main/java/com/trackpack/app/model/trackingmodel/ShipmentTracking.java

```diff
@@ -0,0 +1,40 @@
+package com.trackpack.app.model.trackingmodel;
+
+import com.trackpack.app.model.usermodel.User;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+
+@Data
+@AllArgsConstructor
+public class ShipmentTracking {
+
+    private String id;
+    private String href;
+    private String carrier;
+    private String trackingCode;
+    private String carrierTrackingUrl;
+    private LocalDateTime trackingDate;
```

Comment:
Usage of LocalDateTime doesn't seem right. You'll store date and time without timezone. But:
1) If someone sent you time with timezone, will you lost it?
2) If you need to compare two dates - you should be shure that they're in same zone.
3) How to manage dates with diffent timecodes?
Etc.